include/pi-peps/warnings.h

```diff
@@ -1,83 +1,43 @@
 #ifdef COMPILER_HAS_DIAGNOSTIC_PRAGMA
-#  define DISABLE_WARNINGS                                                     \
```

Comment:
interesting. I didn't notice that. we need to fix this otherwise every time we call `ninja indent` we have this issue. I think the easiest solution would be to put the content of this file into `meson/config/pi-peps/config.h.in` and delete this file. Could you do this?